compiler/rustc_mir/src/transform/remove_noop_landing_pads.rs

```diff
@@ -39,6 +39,7 @@ impl RemoveNoopLandingPads {
                 | StatementKind::StorageDead(_)
                 | StatementKind::AscribeUserType(..)
                 | StatementKind::Coverage(..)
+                | StatementKind::CopyNonOverlapping(..)
```

Comment:
This functions tries to detect cleanup blocks that have no observable side effects after unwinding resumes. The copy non-overlapping could change arbitrary memory location so it doesn't belong there.

Reply:
So I assume that it should be in the set below which return false?